dev.Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM rust:1.49
```

Comment:
It would be better to use the version 1.50, right?

Reply:
I was using the version I have in my system, Its perfectly fine for me to use any non breaking version. My only objection is having locked version (never use latest as version specifier).

Its fine for me if you want to move it to `rust:1.50`

Reply:
Yes, let's use the last release!

Reply:
Sure!